### ISSUE TITLE
correct_prediction写错了吧？

### ISSUE THREAD

**Issue by elias-1** [rockingdingo/deepnlp#2]
在ner里的 ner_model_bilstm.py  correct_prediction= tf.equal(tf.argmax(logits, 1), tf.argmax(targets, 0))  我觉得后面的tf.argmax(targets, 0) 应该修改为（tf.reshape(targets, [-1])）

**Comment by elias-1** [comment]
不过你写了这变量 也没用。。。

**Comment by rockingdingo** [comment]
多谢您，您说的 tf.reshape(targets, [-1]) 这样的写法是正确的，因为现在的batch_size是1，target形状是 [1, time_step] 所以 tf.argmax(targets, 0) 其实也返回 (time_step,) 的向量。我会修改过来的。
